Dockerfile

```diff
@@ -1,45 +1,28 @@
 FROM python:3.8.3-buster
 
-RUN apt-get update \
-    && apt-get install -y \
-        ca-certificates \
-        gcc \
-        git \
-        libev4 \
-        libev-dev \
-        libevent-dev \
-        libxml2-dev \
-        libxslt1-dev \
-        libffi-dev \
-        python-dev \
-        libpq-dev \
-        graphviz \
-        libgraphviz-dev \
-        pkg-config \
-        curl \
-        gosu \
```

Comment:
Why don't we need these anymore?

Reply:
TBH, I don't know.  I spent Friday building and rebuilding this docker image to see if there's anything there that I am missing, but it doesn't appear that we need these libraries.  Once this is deployed to staging we'll see if we need these.